keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -1088,10 +1235,18 @@ type Device struct {
     // +optional
     Attributes []DeviceAttribute
 
-    // TODO for 1.31: define how to support partitionable devices
+    // SharedCapacityConsumed defines the set of shared capacity consumed by
+    // this device.
+    //
+    // Must not have more than 256 entries.
+    //
+    // +listType=atomic
+    // +optional
+    SharedCapacityConsumed []SharedCapacity
 }
 
 const ResourceSliceMaxAttributesPerDevice = 32
+const ResourceSliceMaxSharedCapacityConsumedPerDevice = 256
```

Comment:
Just curious, in the MIG case how many fully partitionable GPUs does this allow in a single `ResourceSlice`?

Reply:
There are 7 shared capacities per GPU, so `floor(256/7) = 36`

Reply:
I've dropped this to 128, per patrick's comment below. It can now only handle. I was also wrong about it being 7 capacities. It is actually 14 because we don't treat the memory blocks as a single "range" value (but rather 8 distinct capacities).

This gives us `floor(128/14) = 9`.

So I can fit all GPUs from in an 8 GPU server in one slice, but would need 2 slices for a server with 16 GPUs.